lib/ontologies_linked_data/models/agents/agent.rb

```
@@ -17,18 +17,24 @@ class Agent < LinkedData::Models::Base
 
+      prevent_serialize_when_nested :usages  
+          
       write_access :creator
       access_control_load :creator
 
       enable_indexing(:agents_metadata)
 
       def embedded_doc
-        "#{self.name} #{self.acronym} #{self.email} #{self.agentType}"
+        {
+          "id": "#{self.id}",
```

RuboCop (1.75.5)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)